### PR TITLE
add MY FTB server back to list

### PR DESCRIPTION
**Pull request by Dream-Master**
Selling kits and everyone on the server get the benefit are not against Eula so far ik.

### CODE REVIEW

File: config/defaultserverlist.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "Test Server GTNH 4 EU": "zeta.gtnewhorizons.com",
     "Prospercraft GTNH Server US": "newhorizonsold.prospercraft.com",
     "StoneLegion.com GTNH (Canada)": "mc.stonelegion.com",
+	"MyFTB GTNH German Server": "newhorizons.myftb.de",
```
Comment (wlhlm) [inline config/defaultserverlist.json:12]: Just to clarify: this domain is different to [the one we remove](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/16821/files#diff-d09fbafc9d969165d4e2718faf79e02b17c6bd63db392f30b749a072295be28dL9). Is that correct?

**Comment by chochem** [comment]
this post is not correct? https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16814#issuecomment-2241762175

File: config/defaultserverlist.json (hunk 1 above)
Comment (Einhornyordle) [reply config/defaultserverlist.json:12]: gtnewhorizons.myftb.de is the right one

Comment (Einhornyordle) [reply config/defaultserverlist.json:12]: yes, just as it was before the removal. It's also on the website if  you open up the "Serveradressen" part https://myftb.de/servers/gt-new-horizons

**Comment by chochem** [comment]
> this post is not correct? [#16814 (comment)](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16814#issuecomment-2241762175)

ok yea apparently it was incorrect or misleading at best
image
jeder spieler = every player on the server gets that

File: config/defaultserverlist.json (hunk 1 above)
Comment (Dream-Master) [reply config/defaultserverlist.json:12]: Ok will correct later

**Review by chochem — APPROVED** [review]

**Pull request merged**